packages/web-runtime/src/components/UserMenu.vue

```diff
@@ -26,64 +25,118 @@
       class="oc-width-auto"
       padding-size="small"
     >
-      <ul class="oc-list">
-        <li class="oc-text-nowrap">
-          <oc-button
-            id="oc-topbar-account-manage"
-            type="router-link"
-            :to="{ path: '/account' }"
-            appearance="raw"
-            gap-size="xsmall"
-            justify-content="left"
-          >
-            <oc-icon name="user" fill-type="line" />
-            <translate>Profile</translate>
-          </oc-button>
-        </li>
-        <li v-for="(n, nid) in menuItems" :key="`user-menu-${nid}`">
-          <oc-button
-            v-if="n.url"
-            type="a"
-            appearance="raw"
-            gap-size="xsmall"
-            justify-content="left"
-            :target="n.target"
-            :href="n.url"
-          >
-            <oc-icon :name="n.icon" />
-            <translate>{{ n.title }}</translate>
-          </oc-button>
-          <oc-button
-            v-else
-            type="router-link"
-            appearance="raw"
-            gap-size="xsmall"
-            justify-content="left"
-            :to="{ path: n.path }"
-          >
-            <oc-icon :name="n.icon" />
-            <translate>{{ n.title }}</translate>
-          </oc-button>
-        </li>
-        <li>
-          <oc-button
-            id="oc-topbar-account-logout"
-            appearance="raw"
-            gap-size="xsmall"
-            justify-content="left"
-            @click="logout"
-          >
-            <oc-icon name="logout-box-r" fill-type="line" />
-            <translate>Log out</translate>
-          </oc-button>
-        </li>
-      </ul>
+      <template #special>
+        <oc-list raw="true" class="oc-pb-l oc-pt-s oc-pl-s oc-pr-s">
```

Comment:
```suggestion
        <oc-list :raw="true" class="oc-pb-l oc-pt-s oc-pl-s oc-pr-s">
```

Otherwise Vue complains about receiving a string and expecting a bool ;)

Reply:
okok i see :D